kafka/coordinator/base.py

```diff
@@ -228,11 +228,16 @@ def _reset_find_coordinator_future(self, result):
         self._find_coordinator_future = None
 
     def lookup_coordinator(self):
-        if self._find_coordinator_future is None:
-            self._find_coordinator_future = self._send_group_coordinator_request()
-
-            self._find_coordinator_future.add_both(self._reset_find_coordinator_future)
-        return self._find_coordinator_future
+        if self._find_coordinator_future is not None:
+            return self._find_coordinator_future
+
+        # If there is an error sending the group coordinator request
+        # then the errback will immediately fire and reset _find_coordinator_future = None
```

Comment:
should this be `_reset_find_coordinator_future`? 

If you're actually trying to use "reset" as a verb, can you pick a different word as it makes it confusing since both `_find_coordinator_future` and `_reset_find_coordinator_future` are used in this block of code...

Reply:
+1